### PR TITLE
Tweaks to chapter 3

### PR DESCRIPTION
**Pull request by jni**
@stefanv have a look especially at fbe81ad, I'm not sure if you removed that formatting because it caused book build problems?

---
This change is [Reviewable](https://reviewable.io/reviews/elegant-scipy/elegant-scipy/288)

### CODE REVIEW

File: markdown/ch3.markdown [hunk 1]
```diff
@@ -8,7 +8,7 @@
 > — William Blake, *The Tyger*
 
 You probably know that digital images are made up of *pixels*. These are
-the light signal *sampled on a regular grid*. When computing
+*samples* of the light signal *on a regular grid*. When computing
```
Comment (stefanv) [inline markdown/ch3.markdown]: "samples of a light signal on a grid" makes less sense than "sampling the light signal on a grid"; either revert or let's discuss.

File: markdown/ch3.markdown [hunk 2]
```diff
@@ -620,7 +621,7 @@ Now we can try it out on the coins image:
 
 ```python
 sobel_mag = ndi.generic_filter(coins, sobel_magnitude_filter, size=3)
-plt.imshow(sobel_mag);
+plt.imshow(sobel_mag, cmap='viridis');
```
Comment (stefanv) [inline markdown/ch3.markdown]: Why is this necessary?

File: markdown/ch3.markdown [hunk 3]
```diff
@@ -658,7 +659,7 @@ another. (*Synapses* are the chemical connections through which neurons
 communicate.) The worm is an awesome example of neural connectivity analysis
 because every worm (of this species) has the same number of neurons (302), and the
 connections between them are all known. This has resulted in the fantastic Openworm
-project [^openworm], which I encourage you to follow.
+project [^openworm], which we encourage you to follow.
```
Comment (stefanv) [inline markdown/ch3.markdown]: maybe "which we encourage you to look up"?

File: markdown/ch3.markdown [hunk 4]
```diff
@@ -1111,12 +1101,12 @@ def build_rag(labels, image):
...
 ```
 
-Here's a few reasons why we think this is a brilliant piece of code:
+Here's a few reasons this is a brilliant piece of code:
 
 - `ndi.generic_filter` iterates over array elements *with their neighbors*.
   (Use `numpy.ndindex` to simply iterate over array indices.)
 - We return "0.0" from the filter function because `generic_filter` requires
-  the filter function to return a float. However, we will ignore the filter
+  the filter function to return a float. However, we will ignore the filter's
```
Comment (stefanv) [inline markdown/ch3.markdown]: filtered?

File: markdown/ch3.markdown [hunk 5]
```diff
@@ -1111,12 +1101,12 @@ def build_rag(labels, image):
     return g
 ```
 
-Here's a few reasons why we think this is a brilliant piece of code:
+Here's a few reasons this is a brilliant piece of code:
```
Comment (stefanv) [inline markdown/ch3.markdown]: are a few reasons

File: markdown/ch3.markdown (hunk 1 above)
Comment (jni) [inline markdown/ch3.markdown]: @stefanv what about the current version? Another alternative is to ignore this altogether, since we move on from it rather quickly. But it's a good opportunity to educate... =\

File: markdown/ch3.markdown (hunk 2 above)
Comment (jni) [inline markdown/ch3.markdown]: Consistency with other plots. If you ask why it's not the default, our style file sets grayscale as the default.

Comment (stefanv) [inline markdown/ch3.markdown]: Oh, gotcha!

File: markdown/ch3.markdown (hunk 1 above)
Comment (stefanv) [inline markdown/ch3.markdown]: Excellent.  I mean, the way we sample is, of course, to integrate over an almost-square CCD element, but... let's not get into the weeds :seedling:

File: markdown/ch3.markdown (hunk 4 above)
Comment (stefanv) [inline markdown/ch3.markdown]: Pinging

File: markdown/ch3.markdown [hunk 6]
```diff
@@ -7,8 +7,11 @@
 >
 > — William Blake, *The Tyger*
 
-You probably know that digital images are made up of *pixels*. These are
-the light signal *sampled on a regular grid*. When computing
+You probably know that digital images are made up of *pixels*. Generally, you
+should not think of these as little squares, but as *point samples* of the light
+signal *measured on a regular grid* [^alvyraysmith].
+
+Further, when computing
```
Comment (stefanv) [inline markdown/ch3.markdown]: computing on images---does that sound right?

File: markdown/ch3.markdown (hunk 4 above)
Comment (jni) [inline markdown/ch3.markdown]: @stefanv look at latest changes. Looks like your browser cache wasn't showing them to you.

Comment (stefanv) [inline markdown/ch3.markdown]: This is a different instance of "filter's".  But since you changed it once before, I'm going to make the modification manually and merge.

**Comment by stefanv** [comment]
@jni Please double check if you are happy with the slight tweak I made in 46967a632061c8d1906c5cc9cbc64c704a5047e6

**Comment by jni** [comment]
I do like it! =)
